ckanext/cloudstorage/fanstatic/scripts/cloudstorage-multipart-upload.js

```diff
@@ -352,13 +364,17 @@ ckan.module('cloudstorage-multipart-upload', function($, _) {
 
         _onFinishUpload: function() {
             var self = this;
+            var data_dict = {
+                'uploadId': this._uploadId,
+                'id': this._resourceId,
+            }
+            if (self._btnClick == 'go-metadata') {
+                data_dict.finish = true
```

Comment:
I don't understand - what is the purpose of calling `cloudstorage_finish_multipart` when we go back to dataset